addons/account/demo/account_demo.py

```diff
@@ -215,11 +220,11 @@ def _get_demo_data_attachment(self):
         cid = self.env.company.id
         ref = self.env.ref
         return ('ir.attachment', {
-            f'{cid}_ir_attachment_bank_statement_1': {
+            f'{cid}_ir_attachment_bank_statement_0': {
```

Comment:
You need to update this file too, it contains all the other lines too

Reply:
I've updated the pdf :)

Reply:
Actually you can remove the file
It needs to be "posted" on all the lines, but the feature is not implemented yet (when you generate the statement it should be in the chatter of all the lines)

Reply:
Done, removed the PDF, the ir.attachment, and the message that posted it.